pkg/svcat/service-catalog/plan.go

```diff
@@ -155,3 +186,57 @@ func (sdk *SDK) RetrievePlanByClassAndPlanNames(className, planName string,
 	}
 	return &searchResults.Items[0], nil
 }
+
+// CreatePlan creates a new plan.
+func (sdk *SDK) CreatePlan(opts CreatePlanOptions,
+) (Plan, error) {
+	if opts.Scope == AllScope {
+		return nil, errors.New("invalid scope: all")
+	}
+	fromPlan, err := sdk.RetrievePlanByName(opts.Name, ScopeOptions{
+		Scope:     opts.Scope,
+		Namespace: opts.Namespace,
+	})
+	if err != nil {
+		return nil, err
+	}
+	if opts.Scope.Matches(NamespaceScope) {
+		var plan = fromPlan.(*v1beta1.ServicePlan)
+		return sdk.createServicePlan(plan, opts.Name, opts.Namespace)
+	}
+	var plan = fromPlan.(*v1beta1.ClusterServicePlan)
+	return sdk.createClusterServicePlan(plan, opts.Name)
+}
+
+func (sdk *SDK) createClusterServicePlan(fromPlan *v1beta1.ClusterServicePlan, name string,
+) (*v1beta1.ClusterServicePlan, error) {
+	var plan = &v1beta1.ClusterServicePlan{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+		Spec: fromPlan.Spec,
+	}
+	plan.Spec.ExternalName = name
+	created, err := sdk.ServiceCatalog().ClusterServicePlans().Create(plan)
+	if err != nil {
```

Comment:
nit: if you need to make more changes to this PR, I recommend altering these lines to be 

```
return sdk.ServiceCatalog().ClusterServicePlans().Create(plan)
```